### DISCUSSION TITLE
How do I use more than one autocomplete input on the same html page?

### DISCUSSION THREAD

**Discussion by sethrw** [TarekRaafat/autoComplete.js#395] [General]
Per the title, when I add a new autoComplete instance in the same js file/same html page the second one doesn't seem to work. How to resolve?

**Comment by trasherdk** [comment]
onBlur of the first, fires the second.

But I'm guessing, as you did not provide any code.

**Comment by anonekama** [comment]
I'm also trying to get this to work but unsure how.

I wanted to try to fire it onfocus, but it didn't seem to work.

`
<input id="id1" onfocus="myfunction(this.id)">
`

```
<script>
      function myfunction(this_id) {
           config.selector = this_id
           var autoCompleteJS = new autoComplete(config);
      }
</script>
```
